src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -7,75 +7,93 @@
       >
         <v-icon>mdi-account-plus-outline</v-icon>
         <span>Start by Adding the Completing Party</span>
       </v-btn>
     </div>
 
-    <div class="btn-panel"  v-if="orgPersonList.length > 0">
-      <v-btn outlined color="primary" @click="addOrgPerson([], IncorporatorTypes.PERSON)"
-        :disabled="showOrgPersonForm" id="btn-add-person"
+    <div class="btn-panel" v-if="orgPersonList.length > 0">
+      <v-btn
+        id="btn-add-person"
+        outlined
+        color="primary"
+        :disabled="showOrgPersonForm"
+        @click="addOrgPerson([], IncorporatorTypes.PERSON)"
       >
         <v-icon>mdi-account-plus</v-icon>
         <span>Add a Person</span>
       </v-btn>
-      <v-btn outlined color="primary" :disabled="showOrgPersonForm" class="spacedButton"
-        @click="addOrgPerson([{ roleType: Roles.INCORPORATOR }], IncorporatorTypes.CORPORATION)" id="btn-add-corp"
+      <v-btn
+        id="btn-add-corp"
+        outlined
+        color="primary"
+        class="spacedButton"
+        :disabled="showOrgPersonForm"
+        @click="addOrgPerson([{ roleType: Roles.INCORPORATOR }], IncorporatorTypes.CORPORATION)"
       >
         <v-icon>mdi-domain-plus</v-icon>
         <span v-if="entityFilter(EntityTypes.BCOMP)">Add a Corporation or Firm</span>
         <span v-if="entityFilter(EntityTypes.COOP)">Add Organization</span>
       </v-btn>
-      <v-btn outlined color="primary"
-        @click="addOrgPerson([{ roleType: Roles.COMPLETING_PARTY }], IncorporatorTypes.PERSON)"
-        :disabled="showOrgPersonForm"  class="spacedButton" v-if="!hasRole(Roles.COMPLETING_PARTY, 1, 'ATLEAST')"
+      <v-btn
+        v-if="!hasRole(Roles.COMPLETING_PARTY, 1, 'ATLEAST')"
         id="btn-add-cp"
+        outlined
+        color="primary"
+        class="spacedButton"
+        :disabled="showOrgPersonForm"
+        @click="addOrgPerson([{ roleType: Roles.COMPLETING_PARTY }], IncorporatorTypes.PERSON)"
       >
         <v-icon>mdi-account-plus-outline</v-icon>
         <span>Add the Completing Party</span>
       </v-btn>
     </div>
 
-    <v-card flat class="people-roles-container" v-if="showOrgPersonForm">
-      <OrgPerson v-show="showOrgPersonForm"
+    <v-card v-if="showOrgPersonForm" flat class="people-roles-container">
+      <OrgPerson
+        v-show="showOrgPersonForm"
         :initialValue="currentOrgPerson"
         :activeIndex="activeIndex"
         :nextId="nextId"
         :existingCompletingParty="completingParty"
         @addEditPerson="onAddEditOrgPerson($event)"
         @removePersonEvent="onRemovePerson($event)"
         @resetEvent="resetData()"
-        @removeCompletingPartyRole="removeCompletingPartyAssignment()" />
+        @removeCompletingPartyRole="removeCompletingPartyAssignment()"
+      />
     </v-card>
 
-    <v-card flat v-if="orgPersonList.length > 0" :disabled="showOrgPersonForm" >
+    <v-card v-if="orgPersonList.length > 0" flat :disabled="showOrgPersonForm" >
```

Comment:
I prefer the conditional first as well :) Easier on the brain for whatever reason hehe